app/javascript/components/modals/meta/meta-styles.scss

```diff
@@ -1,171 +1,184 @@
 @import '~styles/settings.scss';
-$mobile-padding: rem(30px);
+$mobile-padding: rem(42px);
 $desktop-padding: rem(50px);
+$mobile-margin: rem(16px);
 
 .c-modal-meta {
-  padding: $mobile-padding;
-  min-width: rem(320px);
-  min-height: rem(320px);
-  width: 100%;
-  max-width: rem(800px);
-  color: $slate;
-  word-break: break-word;
+  width: calc(100vw - #{$mobile-margin} * 2);
```

Comment:
A different calculation from before that does the same thing? Could this not be on the parent?